prebid-server/pbs-modules/greenbids-real-time-data.md

```diff
@@ -0,0 +1,177 @@
+                  ]
+                }
+              ]
+            }
+          }
+        }
+      }
+    }
+```
+
+### List of module configuration options
+
+- `google-cloud-greenbids-project`: Google Cloud project associated with Greenbids
+- `gcs-bucket-name`: Google Cloud Storage (GCS) bucket used to fetch the artefacts ([ONNX](https://onnx.ai/) model and thresholds `.json`) necessary for prediction
+- `cache-expiration-minutes`: The duration (in minutes) after which cached model and thresholds should be considered expired and refreshed
+- `geo-lite-country-path`: Geolocation capabilities to determine the country from an IP address
+- `onnx-model-cache-key-prefix`: prefix necessary for getting cached ONNX model
+- `thresholds-cache-key-prefix`: prefix necessary for getting cached throttling thresholds
+
+```yaml
+greenbids-real-time-data:
+  google-cloud-greenbids-project: "greenbids-357713"
+  gcs-bucket-name: "greenbids-europe-west1-prebid-server-staging"
+  cache-expiration-minutes: 15
+  geo-lite-country-path: "https://git.io/GeoLite2-Country.mmdb"
+  onnx-model-cache-key-prefix: "onnxModelRunner_"
+  thresholds-cache-key-prefix: "throttlingThresholds_"
+```
+
+### Publisher bid request settings
+
+The activated partner publishers add their configs direclty into `bid-request.json` 
+where they indicate the activation of our module 
+in bid request extenstion `bid-request.ext.prebid.analytics.greenbids` 
+for [Analytics Reporter](https://docs.prebid.org/prebid-server/pbs-modules/greenbids-analytics-reporter.html) and 
+`bid-request.ext.prebid.analytics.greenbids-rtd` for Greenbids RTD Module.
+
+The list of the parameters necessary for RTD module activation is as follows:
+
+| Parameter       | Scope             | Description                                                                                         | Example               | Type         |
+|-----------------|-------------------|-----------------------------------------------------------------------------------------------------|-----------------------|--------------|
+| pbuid           | required          | The Greenbids Publisher ID                                                                          | greenbids-publisher-1 | string       |
+| targetTpr       | required          | Ratio of passing the valid bids [0-1]                                                               | 0.9                   | float        |
+| explorationRate | required          | Ratio of traffic without filtering used for training ML model [0-1] (a value of 0.1 will filter 90% of the traffic) | 0.1                   | float        |
+
+Here's an example of how a PBS partner publisher setup using both Greenbids RTD Module and Greenbids AnalyticsReporter shoudl look like:
+
+```json
+"ext": {
+    "prebid": {
+      "analytics": {
+        // extension for Greenbids Analytics Reporter
+        "greenbids": {
+          "pbuid": "PBUID_FROM_GREENBIDS",
+          "greenbidsSampling": 1
```

Comment:
The value of `1` is a very bad value (it puts our infra on load while don't bringing any value, sending a useless request for each ad call).
```suggestion
          "greenbidsSampling": 0.01
```